### PR TITLE
Dogecoin Turkish Languges translation for  Website

### PR DESCRIPTION
**Pull request by emrullahsaku**
Dogecoin Turkish Languges translation for  Website is completed. Please review and accept my request.

### Description

I can translate in Turkish

### Motivation and Context

I like the Dogecoin. I really would like to contribute the Dogecoin community as a volunteer.

### How Has This Been Tested?

just Turkish Language translate.

### Checklist:

- [ ] GitHub issue linked
- [ ] The necessary translations have been added for all languages
- [ ] Any documentation has been updated accordingly
- [ ] Responsive sizes (Mobile) tested
- [ ] Cross Browser tested if necessary
- [ ] Conflicts resolved

### CODE REVIEW

File: guide.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-    <meta name="description" content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide." />
+    <meta name="description" content="Dogecoin, dünya çapında Shiba Inus tarafından tercih edilen, açık kaynaklı bir eşler arası dijital para birimidir." />
```
Comment (cihanbarisgoren) [inline guide.html]: *Shiba Inular tarafından tercih edilen

File: guide.html [hunk 2]
```diff
@@ -27,9 +27,9 @@
     <meta property="og:type" content="website" />
     <meta property="og:url" content="https://dogecoin.com/" />
     <meta property="og:title" content="Dogecoin| Getting started Guide" />
-    <meta property="og:description" content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide." />
+    <meta property="og:description" content="Dogecoin, dünya çapında Shiba Inus tarafından tercih edilen, açık kaynaklı bir eşler arası dijital para birimidir." />
```
Comment (cihanbarisgoren) [inline guide.html]: *Shiba Inular tarafından tercih edilen

File: index.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-    <meta name="description" content="Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide." />
+    <meta name="description" content="Dogecoin, dünya çapında Shiba Inus tarafından tercih edilen, açık kaynaklı bir eşler arası dijital para birimidir." />
```
Comment (cihanbarisgoren) [inline index.html]: *Shiba Inular tarafından tercih edilen

**Review by cihanbarisgoren — DISMISSED** [review]

**Comment by langerhans** [comment]
Hi, and thank you for the contribution! To be able to merge this you'd need to do the translation work in the i18n files. Check this directory: https://github.com/dogecoin/dogecoin.com/tree/gh-pages/assets/i18n . Copy e.g. en.json as tr.json and make your changes there. See for example #184 on how that works exaclty.

**Comment by emrullahsaku** [comment]
Hi, I am glad to to contribute to Dogecoin community to publicize.
I have created tr.json language and translated.

I have committed on github.

Thanks

**Comment by langerhans** [comment]
Thank you! Now you'd still need to revert the changes to the html files where you replaced the English text with Turkish. Then we can merge it :)

**Comment by emrullahsaku** [comment]
Okay, I reverted the changes to the html files where I replaced the English text with Turkish.
Thank you :)

> Thank you! Now you'd still need to revert the changes to the html files where you replaced the English text with Turkish. Then we can merge it :)

**Comment by langerhans** [comment]
Thanks, I'll merge your changes and then add some fixes for it to shop up correctly. Should be visible later today :)

**Review by langerhans — APPROVED** [review]

**Comment by emrullahsaku** [comment]
> Thanks, I'll merge your changes and then add some fixes for it to shop up correctly. Should be visible later today :)

Thank you. I saw it is live now. Great :)

**Review by Aivolt1 — COMMENTED** [review]
Complete

**Comment by emrullahsaku** [comment]
Hi, I wonder that can my name added for contributors page. Thanks

**Pull request merged**